### DISCUSSION TITLE
输出似乎是8bit的？

### DISCUSSION THREAD

**Discussion by op200** [Blinue/Magpie#804] [General]
刚用这个软件，不太清楚功能，我发现输出时，颜色渐变区会出现明显的色带，是不是8bit量化导致的？

**Comment by Blinue** [comment]
处理时每个像素表示为 32 位。有色带很可能是窗口本身就有色带，放大后更加明显而已。Deband 效果可以去色带。

**Reply by op200** [reply]
怪了嘞，我重启软件之后就没色带了qwq
